### PR TITLE
Implemented Unrolled list and updated corresponding readme.md

### PR DESCRIPTION
**Pull request by imsushant12**
Added unrolled linked list implementation and updated corresponding readme.md 

ASSIGNED ISSUE:
link : https://github.com/Algo-Phantoms/Algo-Tree/issues/233

### CODE REVIEW

**Comment by rudrakshi99** [comment]
@imsushant12 Remove this - added sorted & unsorted ziz-zag array and modified related readme.md

File: Code/C++/unrolled_linked_list.cpp [hunk 1]
```diff
@@ -0,0 +1,86 @@
+#include <bits/stdc++.h>
```
Comment (rudrakshi99) [inline Code/C++/unrolled_linked_list.cpp:1]: add a brief explanation to the algo.

File: Code/C++/unrolled_linked_list.cpp [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+using namespace std;
+
+#define N 5
+
+//Unrolled Linked List Node
+struct node
```
Comment (rudrakshi99) [inline Code/C++/unrolled_linked_list.cpp:7]: use `class`  instead of  `struct`

File: Code/C++/unrolled_linked_list.cpp [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+
+    return 0;
+}
+
+/*
+    Test Case -
+
+    Output ;
+    1  2  3  4  5  6  7  8  9  10  11  12
+
```
Comment (rudrakshi99) [inline Code/C++/unrolled_linked_list.cpp:84]: add atleast 2 test cases and complexities.

**Review by rudrakshi99 — CHANGES_REQUESTED** [review]

File: Code/C++/unrolled_linked_list.cpp (hunk 3 above)
Comment (imsushant12) [reply Code/C++/unrolled_linked_list.cpp:98]: done

File: Code/C++/unrolled_linked_list.cpp (hunk 1 above)
Comment (imsushant12) [reply Code/C++/unrolled_linked_list.cpp:1]: done

File: Code/C++/unrolled_linked_list.cpp (hunk 2 above)
Comment (imsushant12) [reply Code/C++/unrolled_linked_list.cpp]: done

**Review by imsushant12 — COMMENTED** [review]
Please view the changes

File: Code/C++/unrolled_linked_list.cpp [hunk 4]
```diff
@@ -0,0 +1,103 @@
...
+    fourth->ptr = NULL;
+
+    display(head);
+
+    return 0;
+}
+
+/*
+    Complexity:
+    Time complexity : O(n)
```
Comment (rudrakshi99) [inline Code/C++/unrolled_linked_list.cpp:93]: add input also

**Review by rudrakshi99 — CHANGES_REQUESTED** [review]

**Comment by rudrakshi99** [comment]
@imsushant12 remove the entry for sorted & unsorted ziz-zag array in the readme for this PR.

File: Code/C++/unrolled_linked_list.cpp (hunk 4 above)
Comment (imsushant12) [reply Code/C++/unrolled_linked_list.cpp:93]: Done. Please review it

**Pull request merged**